src/client/mds_client.cpp

```diff
@@ -913,14 +923,18 @@ LIBCURVE_ERROR MDSClient::GetOrAllocateSegment(bool allocate,
 
         auto statuscode = response.statuscode();
         switch (statuscode) {
+            case StatusCode::kParaError:
```

Comment:
recheck this part of the code https://github.com/opencurve/curve/blob/aa0e97bd764c45c598dfa0821303cd0de54d08b5/src/client/splitor.cpp#L201, modification in here may cause an error.

Reply:
> recheck this part of the code
> 
> https://github.com/opencurve/curve/blob/aa0e97bd764c45c598dfa0821303cd0de54d08b5/src/client/splitor.cpp#L201
> 
> , modification in here may cause an error.

yeap, the function should return `LIBCURVE_ERROR::FAILED` instead of `LIBCURVE_ERROR::NOTEXIST` and `LIBCURVE_ERROR::PARAM_ERROR`.